src/core/lib/surface/init.cc

```diff
@@ -24,6 +24,7 @@
 #include <grpc/fork.h>
 #include <grpc/grpc.h>
 #include <grpc/impl/channel_arg_names.h>
+#include <grpc/support/log.h>
 #include <grpc/support/port_platform.h>
 #include <grpc/support/sync.h>
```

Comment:
Some of the cc files are using gpr_log_verbosity_init() functions.

So I removed the #include <grpc/support/log.h> from all headers and put it selectively only in the cc files that used gpr_log_verbosity_init()